services/app-web/src/pages/StateSubmission/RateDetails/RateDetailsSchema.ts

```diff
@@ -11,6 +11,9 @@ Yup.addMethod(Yup.date, 'validateDateFormat', validateDateFormat)
 
 const SingleRateCertSchema = (_activeFeatureFlags: FeatureFlagSettings) =>
     Yup.object().shape({
+        ratePreviouslySubmitted: Yup.string().defined(
```

Comment:
Add feature flag here - should only be checking for these fields on V2 path - behind rate edit unlock or linked rates feature flag